docs/integrations/buy-online-pick-up-in-store/frequently-asked-questions.mdx

```diff
@@ -26,3 +26,7 @@ Order Refunds do not affect order consignments.
 When an order is edited, the order consignment will be modified to reflect these changes. For example, this includes changes to shipping addresses or line item quantities, as well as the addition or removal of line items.
 
 An order can be edited at any time of the fulfillment lifecycle; this includes orders that have already been fulfilled.
+
+**As an app developer, how can I ensure my app is compatible with multi-location inventory?**
+
+Apps that are [compatible with multi-location inventory](https://support.bigcommerce.com/s/article/Multi-Location#app-support) must use the [Inventory API](/docs/store-operations/catalog/inventory-adjustments), not the [V3 Catalog API](/docs/rest-catalog), to adjust inventory in a store.
```

Comment:
```suggestion
Apps that are [compatible with multi-location inventory](https://support.bigcommerce.com/s/article/Multi-Location#app-support) must use the [Inventory API](/docs/store-operations/catalog/inventory-adjustments), not the [V3 Catalog API](/docs/rest-catalog), to adjust inventory in a store. For more information, see the [Inventory overview - Breaking changes to Catalog API](/docs/store-operations/catalog/inventory-adjustments#breaking-changes-to-catalog-api) article.
```

Reply:
@jcnich I got some feedback that my doc could be explained better. I added a link to the https://developer.bigcommerce.com/docs/store-operations/catalog/inventory-adjustments#breaking-changes-to-catalog-api page. What do you think?

Reply:
I think that works, although I would probably just link to the general Inventory Overview (https://developer.bigcommerce.com/docs/store-operations/catalog/inventory-adjustments) rather than specifically to the "breaking changes" section, just to avoid any confusion.

Reply:
Sounds good